src/java/org/apache/cassandra/db/memtable/AbstractMemtable.java

```diff
@@ -204,9 +204,9 @@ public CommitLogPosition commitLogLowerBound()
             return AbstractMemtable.this.getCommitLogLowerBound();
```

Comment:
unrelated, but `import org.apache.cassandra.db.lifecycle.LifecycleTransaction;` is unused up there ^

Reply:
Removed